chapter/7/langs-consistency.md

```diff
@@ -6,17 +6,18 @@ by: "James Larisch"
 
-  As Aviral discussed in the previous section, many computer scientists have done a lot of thinking about the nature of distributed systems problems. As such, we realize that it's impossible to completely emulate the behavior of a single computational machine using multiple machines. For example, the network is simply not as reliable as, say, memory - and waiting for responses can result in untimeliness for the application's user base. After discussing the Consistency/Availability/Partition-tolerance theorem, Section 6 discussed how we can drill down into the CAP pyramid and choose the necessary and unnecessary properties of our systems. As stated, we can't perfectly emulate a single computer using multiple machines, but once we accept that fact and learn to work with it, there are plenty of things we *can* do!
+As Aviral discussed in the previous section, many computer scientists have done a lot of thinking about the nature of distributed systems problems. As such, we realize that it's impossible to completely emulate the behavior of a single computational machine using multiple machines. For example, the network is simply not as reliable as, say, memory - and waiting for responses can result in untimeliness for the application's user base. After discussing the Consistency/Availability/Partition-tolerance theorem, Section 6 discussed how we can drill down into the CAP pyramid and choose the necessary and unnecessary properties of our systems. As stated, we can't perfectly emulate a single computer using multiple machines, but once we accept that fact and learn to work with it, there are plenty of things we *can* do!
 
 ## The Shopping Cart
-  Let's bring all this theorem talk back to reality. Let's say you're working at a new e-commerce startup, and you'd like to revolutionize the electronic shopping cart. You'd like to give the customer the ability to do the following:
-  1. Log in to the site and add a candle to the cart while traveling in Beijing.
-  1. Take a HyperLoop (3 hours) from Beijing to Los Angeles.
-  1. Log back in, remove the candle from the cart, and add a skateboard.
-  1. Take another HyperLoop train from Los Angeles to Paris (5 hours).
-  1. Log back into the site, add another skateboard, and checkout.
+Let's bring all this theorem talk back to reality. Let's say you're working at a new e-commerce startup, and you'd like to revolutionize the electronic shopping cart. You'd like to give the customer the ability to do the following:
+
+1. Log in to the site and add a candle to the cart while traveling in Beijing.
+2. Take a HyperLoop (3 hours) from Beijing to Los Angeles.
+3. Log back in, remove the candle from the cart, and add a skateboard.
+4. Take another HyperLoop train from Los Angeles to Paris (5 hours).
+5. Log back into the site, add another skateboard, and checkout.
```

Comment:
fun fact: markdown will automatically number things in order. you can put any number there :)